src/CMakeLists.txt

```diff
@@ -637,9 +637,14 @@ install(TARGETS ceph-mon DESTINATION bin)
 if(NOT ALLOCATOR STREQUAL "jemalloc")
   set(disable_jemalloc "DISABLE_JEMALLOC=1")
 endif()
+set(EXTRA_CXXFLAGS -fPIC -Wno-unused-variable -DNDEBUG)
```

Comment:
could you add a prefix for `EXTRA_CXXFLAGS`, so it is `ROCKSDB_EXTRA_CXXFLAGS`?

so this name has less chance to collide with other (future) names.